chain/network/src/test_utils.rs

```diff
@@ -281,12 +282,17 @@ impl MockNetworkAdapter {
     }
 }
 
+pub fn make_ibf_routing_pool() -> Addr<RoutingTableActor> {
+    SyncArbiter::start(1, move || RoutingTableActor::default())
```

Comment:
How do we stop the thread we spawn here?

Reply:
I created it the same way as `
    edge_verifier_pool: Addr<EdgeVerifier>,
`.
We don't stop `EdgeVerifier` either. Maybe we should stop both?

Reply:
Yes

Reply:
Done
@bowenwang1996 Can you check that the logic for stopping is correct?